guacamole/src/main/webapp/translations/de.json

```diff
@@ -125,8 +125,15 @@
 
     },
 
+    "DATA_SOURCE_DEFAULT" : {
+        "NAME" : "Standard (XML)"
+    },
+
     "FORM" : {
 
+        "FIELD_PLACEHOLDER_DATE" : "DD.MM.YYYY",
```

Comment:
This will not do what you expect. The placeholder from `en.json` is in exactly the format expected by the field (YYYY-MM-DD). If that format is not followed by the field contents, the data will be rejected by the web service.

As this is also the exact format used by the HTML5 date field, this has the effect of providing a fallback. If the date field is supported by the browser, the user sees a date picker, otherwise they see helpful information dictating how to format the date.

This should not be translated to the ideal format for German dates. It should be translated to the German method for describing the YYYY-MM-DD format. If Y's, M's, and D's are used in German, too, then nothing needs to be changed.